drivers/hmis/spec/models/hmis/project_group_spec.rb

```diff
@@ -0,0 +1,51 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Hmis::ProjectGroup, type: :model do
```

Comment:
I'm probably missing something, but how does this file check the project group?

Reply:
🤦  didn't write this test yet... thanks. (its copy-pasted from another file). I forgot about this so its not quite ready for review. Everything else should be though!

Reply:
Tests added!